google/cloud/bigtable/instance_admin_client.h

```diff
@@ -325,6 +325,14 @@ class InstanceAdminClient {
                     google::longrunning::GetOperationRequest const& request,
                     grpc::CompletionQueue* cq) = 0;
   //@}
+
+ private:
+  friend class InstanceAdmin;
```

Comment:
Do you need the `friend`?

Reply:
yes. https://github.com/googleapis/google-cloud-cpp/blob/a62ea5760134a6ec48959777a21943bf95f2b590/google/cloud/bigtable/instance_admin.h#L148